### ISSUE TITLE
【论文复现】 取Linear层的in_features报错，其他类型的层可以

### ISSUE THREAD

**Issue by BADBADBADBOY** [PaddlePaddle/Paddle#37646]
model = resnet50()
for (i,layer) in enumerate(model.sublayers()):
    if type(layer)==nn.Conv2D:
        print(layer)
        print(layer._stride)
    elif type(layer)==nn.BatchNorm2D:
        print(layer)
        print(layer._num_features)
    elif type(layer)==nn.Linear:
        print(layer)
        print(layer._in_features)

paddle版本：2.2.0

**Comment by ForFishes** [comment]
您好。``nn.Linear``这个类没有``_in_features``的这个属性。如果要打印权重的shape信息，可以用如下方法：
``
print(layer.weight.shape[0], layer.weight.shape[1])
``
